gammapy/cube/background.py

```diff
@@ -22,38 +22,38 @@ def make_map_background_irf(pointing, ontime, bkg, geom, n_integration_bins=1):
         Background rate model
     geom : `~gammapy.maps.WcsGeom`
         Reference geometry
-    n_integration_bins : int
-        Number of bins per energy bin in integration
+    integrate : bool
+        Whether to integrate the background model over the energy axes.
 
     Returns
     -------
     background : `~gammapy.maps.WcsNDMap`
         Background predicted counts sky cube in reco energy
     """
-    energy_axis = geom.axes[0]
-    ebounds = energy_axis.edges * energy_axis.unit
+    energy_axis = geom.get_axis_by_name("energy")
+
 
     # Compute FOV coordinates; at the moment assume symmetric background model
     # TODO: implement FOV coordinates properly
-    map_coord = geom.get_coord()
+    map_coord = geom.to_image().get_coord()
     fov_lon = map_coord.skycoord.separation(pointing)
     fov_lat = Angle(np.zeros_like(fov_lon), fov_lon.unit)
 
-    if n_integration_bins == 0:
-        energy_reco = map_coord[energy_axis.name] * energy_axis.unit
+    if not integrate:
```

Comment:
Is there any use case for `integrate = False`?

Reply:
Good remark.
 A priori, background maps are always integrated quantities defined between energy edges and not at bin centers. In that sense, the log-log trapezoidal integral is the most relevant case. 
The `integrate = False` is therefore probably irrelevant here.